README.md

```diff
@@ -10,6 +10,7 @@ What does it do?
 - Prevent merge marker commits.
 - Automatically bump versions when starting a release or hotfix. Versions are generated, written to file and committed.
 - Automatically specify tag messages.
+- Automatically creates a CHANGELOG file if USE_CHANGELOG=true.
```

Comment:
Maybe be a bit less specific about the technical side and simple state what it does?

I think "Automatically create a change-log." would suffice.